src/commands/kv/bulk/mod.rs

```diff
@@ -1,37 +1,50 @@
-extern crate base64;
+mod delete;
+mod write;
 
-use cloudflare::framework::apiclient::ApiClient;
-
-use std::fs;
-use std::fs::metadata;
-use std::path::Path;
+pub use delete::delete;
+pub use write::write;
 
+use cloudflare::endpoints::workerskv::delete_bulk::DeleteBulk;
 use cloudflare::endpoints::workerskv::write_bulk::KeyValuePair;
 use cloudflare::endpoints::workerskv::write_bulk::WriteBulk;
+use cloudflare::framework::apiclient::ApiClient;
 
 use crate::commands::kv;
+use crate::message;
 use crate::settings::global_user::GlobalUser;
 use crate::settings::project::Project;
-use crate::terminal::message;
 
 const MAX_PAIRS: usize = 10000;
 
-pub fn write_json(
+fn delete_bulk(
```

Comment:
I think it would be good to move this to delete.rs and make it public, given that bucket will use it in the future :)